variants/heltec_wireless_tracker_V1_1/platformio.ini

```diff
@@ -0,0 +1,12 @@
+[env:heltec-wireless-tracker-V1-1]
```

Comment:
Let's make this platformio variant name remain 'heltec-wireless-tracker' for v1.1? Otherwise, this breaks all of the previous CI/CD work as well as the meshtastic API for the web flasher.

Reply:
yeah we can do that, i just hope they won't make another version.